files/nginx.conf

```diff
@@ -1,7 +1,7 @@
 # Based on http://sabre.io/baikal/install/
 
 server {
-  listen 80;
```

Comment:
Based on <https://trac.nginx.org/nginx/ticket/345>, this would disable IPv4 support though, right? Please leave that line in there, so that both IPv4 and IPv6 are enabled.